api/repository.go

```diff
@@ -100,7 +101,8 @@ func (find *RepositoryFind) String() string {
 
 // RepositoryPatch is the API message for patching a repository.
 type RepositoryPatch struct {
-	ID int `jsonapi:"primary,repositoryPatch"`
+	ID     int `jsonapi:"primary,repositoryPatch"`
```

Comment:
Should this be int pointer as well?

Reply:
If it is a pointer, it is easy to cause the where clause to be empty and update all the repositories.

Reply:
Oh, I see what you mean. Make sense.

Reply:
In that case, the store module should throw an error.

It's the same as our require.where rule

Reply:
Done. PTAL @tianzhou